lib/rubocop/cop/base.rb

```diff
@@ -137,6 +137,7 @@ def external_dependency_checksum
       end
 
       def self.inherited(subclass)
+        super
```

Comment:
😊

Reply:
@fatkodima Marc-André is smiling for a reason.
There's no `super` for a class with no parent.

Reply:
Actually I still like adding it, and I was surprise I didn't in the first place.